docs/understand/data-structure.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Data Structure
+parent: Understanding lakeFS
+description: Understand the data structure in lakeFS
+---
+
+# How Does lakeFS Store Your Data
+lakeFS being a data versioning engine, requires the ability to save multiple versions of the same object. As a result, lakeFS stores objects in the object store
+in an unconventional way.
+This might cause confusion when trying to understand where our data is actually being stored. This page will try to shed a light on this subject.
+
+## lakeFS Repository Namespace Structure
+lakeFS stores repository data and metadata under the repository's namespace.
```

Comment:
Explain that a storage namespace is a path in an object store that's dedicated to the operation of that repository